setup.py

```diff
@@ -177,19 +178,22 @@ def get_build_options():
     # FIXME it is a wrong place for this dependency
     if not no_dist:
         include_dir_plat.append(mpi_root + '/include')
-    using_intel = os.environ.get('cc', '') in ['icc', 'icpc', 'icl', 'dpcpp']
+    using_intel = os.environ.get('CC', '') in [
+        'icc', 'icpc', 'icl', 'dpcpp', 'icx', 'icpx']
     eca = ['-DPY_ARRAY_UNIQUE_SYMBOL=daal4py_array_API',
            '-DD4P_VERSION="' + d4p_version + '"', '-DNPY_ALLOW_THREADS=1']
     ela = []
 
-    if using_intel and IS_WIN:
-        include_dir_plat.append(
-            jp(os.environ.get('ICPP_COMPILER16', ''), 'compiler', 'include'))
-        eca += ['-std=c++17', '-w', '/MD']
+    if using_intel:
+        if IS_WIN:
+            include_dir_plat.append(
+                jp(os.environ.get('ICPP_COMPILER16', ''), 'compiler', 'include'))
+            eca += ['-std=c++17', '/MD']
```

Comment:
```suggestion
            eca += ['-std=c++17', '/MT']
```

Looks like this should be changed for resolved compatibility with updated MKLGPUFPK

Reply:
Let's check it properly on latest MKLGPUFPK